### ISSUE TITLE
Error configuration for CI of caffe2

### ISSUE THREAD

**Issue by Stonesjtu** [pytorch/pytorch#6169]
output: (https://ci.pytorch.org/jenkins/job/caffe2-builds/job/py2-cuda8.0-cudnn6-ubuntu16.04-test/2157/console)
```
07:33:29 Downloading from https://s3.amazonaws.com/download.caffe2.ai/models/shufflenet/predict_net.pb
07:33:29 Abort: Could not download model. [HTTP Error] 404: Not Found.
```
PS. I think the separate trigger for `pytorch` and `caffe2` does not work correctly. #5936 always fires the tests of caffe2

**Comment by ezyang** [comment]
> PS. I think the separate trigger for pytorch and caffe2 does not work correctly. #5936 always fires the tests of caffe2

Oh, I know what the problem is. PR here: https://github.com/pytorch/pytorch/pull/6177

**Comment by soumith** [comment]
closed via #6177
